src/main/java/uk/gov/hmcts/reform/unspec/handler/CreateClaimCallbackHandler.java

```diff
@@ -31,23 +39,30 @@
 public class CreateClaimCallbackHandler extends CallbackHandler {
 
     private static final List<CaseEvent> EVENTS = Collections.singletonList(CREATE_CASE);
+    public static final String CONFIRMATION_SUMMARY = "<br />Follow these steps to serve a claim:"
+        + "\n* <a href=\"%s\" target=\"_blank\">[Download the sealed claim form]</a> (PDF, %sKB)"
+        + "\n* Send the form, particulars of claim and "
+        + "<a href=\"%s\" target=\"_blank\">a response pack</a> (PDF, 266 KB) to the defendant by %s"
+        + "\n* Confirm service online within 21 days of sending the form, particulars and response pack, before"
+        + " 4pm if you're doing this on the due day";
 
-    private final ObjectMapper mapper;
     private final String responsePackLink;
-
-    public CreateClaimCallbackHandler(
-        ObjectMapper mapper,
-        @Value("${unspecified.response-pack-url}") String responsePackLink
-    ) {
-        this.mapper = mapper;
+    private final SealedClaimFormGenerator sealedClaimFormGenerator;
+    private final CaseDetailsConverter caseDetailsConverter;
+
+    public CreateClaimCallbackHandler(CaseDetailsConverter caseDetailsConverter,
+                                      SealedClaimFormGenerator sealedClaimFormGenerator,
+                                      @Value("${unspecified.response-pack-url}") String responsePackLink) {
```

Comment:
I'd say that we should have `@Value()` annotations only in configuration classes, thoughts?